examples/HowTo.md

```diff
@@ -21,42 +26,56 @@ module Main (main) where
 
 
 import Control.Monad.IO.Class (MonadIO, liftIO)
-import Data.Monoid ((<>))
+import Data.Semigroup ((<>))
 import Data.Text (Text)
 import System.Wlog (WithLoggerIO, buildAndSetupYamlLogging, logError,
-                    logInfo,productionB, usingLoggerName)
+                    logInfo, productionB, usingLoggerName)
 
-import qualified Data.Text as T
-import qualified Data.Text.IO as TIO
+import qualified Data.Text as Text
+import qualified Data.Text.IO as TextIO
 
 ```
 
 ## Adding a new logger
 
 First of all, you may ask what are we doing? What the hell is logger?
 
-**Logger** is the [output handler](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LogHandler.html#t:LogHandler) labeled by the unique [logger name](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LoggerName.html#t:LoggerName).
+**Logger** is the [output handler](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LogHandler.html#t:LogHandler)
+labeled by the unique [logger name](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LoggerName.html#t:LoggerName).
+
+When you log new messages you need to specify logger which is responsible for logging this message.
+Fortunately, log-warper has monadic interfaces and logger will be taken explicitly from current context
+if possible, thus you need to specify logger only once for some block of actions.
+Don't worry, this will be covered later!
 
-To add a new logger for your needs you'll have to consider several things.
-Firstly, you should consider choosing some reasonable name for the logger you creating according to the actions that it would cover with logging. From now on let's assume that you need to have `new-logger` in your project.
+So, if you're adding a new logger for your needs you'll have to consider several things.
+Firstly, you should choose some reasonable name for the logger you are creating
+according to the actions that it would cover with logging. From now on let's assume
+that you need to have the logger with the name `new-logger` in your project.
 
 ### Configuration file
 
-To configure your logger you will need configuration `.yaml` file.
-In our example we will use very simple one with the minimal settings needed for the file to work:
+There are two ways of configuring the logger you creating. One option
+is to configure everything directly in the code, which will make your
+code less convenient. Here we will use another configuration possibility —
```

Comment:
Well, code can't be less or more convenient. It's the process of writing code which can be less or more convenient..